runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/ClueScrollPlugin.java

```diff
@@ -434,6 +452,39 @@ public void onItemContainerChanged(final ItemContainerChanged event)
 		}
 	}
 
+	private void setActiveClue(Integer itemId)
+	{
+		ClueScroll clueScroll = findClueScroll(itemId);
+
+		if (clueScroll != null)
+		{
+			clueItemId = itemId;
+			updateClue(clueScroll);
+		}
+	}
+
+	private Set<Integer> getClueItemIdsInInventory(ItemContainer inventoryContainer)
+	{
+		// Remove clues no longer in inventory
+		clueItemIdsInInventory.removeIf(clue -> !inventoryContainer.contains(clue));
+
+		// Add new clues
+		Item[] inventory = inventoryContainer.getItems();
+		Set<Integer> cluesFound = new HashSet<>();
+
+		for (Item item : inventory)
+		{
+			int invItemId = item.getId();
+			// If paramID 623 returns a dbrow, the invItemId is a valid Clue Scroll
+			if (client.getItemDefinition(invItemId).getIntValue(623) != -1)
```

Comment:
This param should become a documented constant in ParamID.

Reply:
My understanding is that param 623 points to a 'clueScrollRow', which will point to a 'clueSolutionRow'. So this is a good way to check if something is a clue.
Any suggestions would be appreciated for what the constant would be called in https://github.com/runelite/runelite/blob/182b8d7822d088b4206acffbf8546aa8df22cc21/runelite-api/src/main/java/net/runelite/api/ParamID.java
Would this be included in this PR or separate?

Reply:
Added new ParamID. Would like to give it a better name though